src/components/user/LastUserWidget.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div>
+    <p class="title text-lg-center">
+      Users
+    </p>
+    <v-text-field
+      label="Search"
+      @input="(q)=>fetchUsers({q})"
+    />
+    <div v-if="loading" class="text-xs-center mt-5">
+      <v-progress-circular
+        :width="1"
+        color="red"
+        :size="70"
+        indeterminate
+      />
+    </div>
+    <div v-for="user in userList" v-else :key="user.id" class="avatars">
+      <router-link :to="{name: 'user', params: {id: user.id}}">
+        <v-avatar class="d-inline-flex ma-2" color="indigo">
+          <img
+            :src="user.profile_image"
+            alt="John"
+          >
+        </v-avatar>
+      </router-link>
+    </div>
+  </div>
+</template>
+<script>
+import { mapActions, mapGetters } from 'vuex'
+
+export default {
+  name: 'LastUserWidget',
+  computed: {
+    ...mapGetters({
+      userList: 'user/getList',
+      loading: 'user/getLoading'
+    })
+  },
+  mounted () {
```

Comment:
Also use `created` here and check if there are same cases.

Reply:
Changed to `created`